src/doc/trpl/structs.md

```diff
@@ -87,3 +87,32 @@ fn main() {
     point.y = 6; // this causes an error
 }
 ```
+
+# Update syntax
+
+A `struct` can include `..` to indicate that you want to update just some
+values of a struct. For example:
```

Comment:
I've added a note about the copy in the second version, but considering this is more of a codegen issue, I think describing the semantics intended is still right here.